samples/sample_encode/src/sample_encode.cpp

```diff
@@ -1411,11 +1411,11 @@ mfxStatus ParseInputString(msdk_char* strInput[], mfxU8 nArgNum, sInputParams* p
         return MFX_ERR_UNSUPPORTED;
     }
 
-    if (pParams->nLADepth && (pParams->nLADepth < 10))
+    if (pParams->nLADepth && (pParams->nLADepth > 100))
```

Comment:
Did you intend to write (mind `!` and `||`):
`if (!InputParams.nLADepth || (InputParams.nLADepth > 100))`

Mind that after your change the condition @ line 2613 stop to make sense. I mean this one:
`if ((InputParams.nLADepth != 1) || (!InputParams.nMaxSliceSize))`

Please, check its correctness and update/remove as needed. Question: does `-mss` require `nLADepth == 1` or it can work with other depths as well?